spec/abi.dd

```diff
@@ -238,11 +244,11 @@ $(H3 $(LNAME2 name_mangling, Name Mangling))
         $(GNAME TemplateArgX):
             $(B T) $(GLINK Type)
             $(B V) $(GLINK Type) $(GLINK Value)
-            $(B S) $(GLINK LName)
+            $(B S) $(GLINK SymbolName)
+            $(B S) $(GLINK MangledName)
```

Comment:
Yeah, noticed this too last night, see https://issues.dlang.org/show_bug.cgi?id=14591#c7